content/en/docs/tasks/administer-cluster/topology-manager.md

```diff
@@ -262,9 +262,14 @@ In the above two cases of the `Guaranteed` pod, the `none` CPU Manager policy wo
 
 In the case of the `BestEffort` pod, the `static` CPU Manager policy would send back the default topology hint as there is no CPU request and the Device Manager would send back the hints for each of the requested devices.
 
-Using this information the Topology Manager calculates the optimal hint for the pod and stores this information, which will be used by the Hint Providers when they are making their resource assignments. 
+Using this information the Topology Manager calculates the optimal hint for the pod and stores this information, which will be used by the Hint Providers when they are making their resource assignments.
 
 ### Known Limitations
 1. The maximum number of NUMA nodes that Topology Manager allows is 8. With more than 8 NUMA nodes there will be a state explosion when trying to enumerate the possible NUMA affinities and generating their hints.
+2. The default scheduler is not topology-aware, so it is possible to be scheduled on a node but then fail admission on the node due to the Topology Manager.
+However, an [out-of-tree scheduler plugin](https://github.com/kubernetes-sigs/scheduler-plugins/tree/master/pkg/noderesourcetopology) mitigates this problem and allows optimal scheduling by taking resource topology into consideration.
+
+In order to achieve this, the plugin needs to subscribe to a `NodeResourceTopology` Custom Resource endpoint.

```

Comment:
This should be part of the 2nd point. This [renders](https://deploy-preview-30806--kubernetes-io-main-staging.netlify.app/docs/tasks/administer-cluster/topology-manager/) as a separate paragraph in the `Known Limitations` section and could possibly give the impression that we have moved to another topic.